dev/ansible/05_presto_create_tables.yml

```diff
@@ -1,25 +1,25 @@
 - name: Presto healthchecks pass
   uri:
-    url: http://localhost:8500/v1/health/checks/presto?filter={{ '(Status=="passing")' | urlencode }}
+    url: http://localhost:8500/v1/health/checks/presto
     method: GET
     return_content: yes
     status_code: 200
     body_format: json
   register: result_presto
   retries: 15
   delay: 15
-  until: result_presto.json | length == 3
+  until: '"critical" not in result_presto | string'
```

Comment:
I suggested the same solution Fredrik uses here a little while back. You mentioned you want it explicitly checking every time, but I want to say I still agree with what Fredrik has done, because it is more generic and doesn't crash immediately if we decide to add more checks. Less maintenance down the road.

Reply:
My point here, this check will pass if status `warning`, but not sure that status `warning` will satisfy the requirements to continue

Reply:
ok.. So the problem with keeping it as is is that standalone and cluster will have a different number of tests. I generally would like to test for critical rather than number of passing tests.

Reply:
I'll mark this as resolved for now.